taccsite_cms/static/site_cms/css/src/core-cms.css

```diff
@@ -15,4 +15,6 @@
 @import url("./_imports/components/django-cms-video.css");
 @import url("./_imports/components/lightgallery.css");
 
+@import url("./_imports/generics/pseudo-elements.css");
```

Comment:
Please move generics between settings and elements, not after components.

This will ensure that element styles can override generic styles, as expected.

Reference: [ITCSS (internal doc)](https://confluence.tacc.utexas.edu/x/IAA9Cw)

Reply:
> Please move generics between settings and elements, not after components.
> 
> This will ensure that element styles can override generic styles, as expected.
> 
> Reference: [ITCSS (internal doc)](https://confluence.tacc.utexas.edu/x/IAA9Cw)

https://github.com/TACC/Core-CMS/pull/738/commits/ea61beab174db9f9c90d8abc1091b4a7481c3906